ngraph/frontend/onnx_import/CMakeLists.txt

```diff
@@ -73,11 +71,8 @@ install(TARGETS onnx_importer EXPORT ngraphTargets
         LIBRARY DESTINATION ${NGRAPH_INSTALL_LIB} COMPONENT ngraph)
 
 install(DIRECTORY ${ONNX_IMPORT_INCLUDE_DIR}/onnx_import
-    DESTINATION ${ONNX_INSTALL_INCLUDE}
-    COMPONENT ngraph
-    FILES_MATCHING
-        PATTERN "*.hpp"
-        PATTERN "*.h"
-)
+        DESTINATION ${FRONTEND_INSTALL_INCLUDE}
+        COMPONENT ngraph_dev
```

Comment:
ngraph_dev contains only '.hpp' files, there is no ".so" libraries. Is it expected? Usual debian 'dev' packages have headers to include and libraries (or symlinks) to link with

Reply:
proper name would be "ngraph-headers", dev here does not correlate with dev in debian :)

Reply:
proper name would be "ngraph-headers", dev here does not correlate with dev in debian :)